stubs/qrbill/qrbill/bill.pyi

```diff
@@ -0,0 +1,159 @@
+#       we can choose to refine in the future, e.g. using a Protocol
+_SvgDrawing: TypeAlias = Any  # svgwrite.Drawing
+_SvgGroup: TypeAlias = Any  # svgwrite.container.Group
+
+# NOTE: Eventually we may want to consider replacing this with typed dicts, even
+#       if that means disallowing non-dict arguments. It should allow anything
+#       that is valid to pass into `Address.create`.
+_AddressDict: TypeAlias = Mapping[str, str | None]
+
+IBAN_ALLOWED_COUNTRIES: list[str]
+QR_IID: dict[str, int]
+AMOUNT_REGEX: str
+MM_TO_UU: float
+BILL_HEIGHT: int
+RECEIPT_WIDTH: str
+PAYMENT_WIDTH: str
+MAX_CHARS_PAYMENT_LINE: int
+MAX_CHARS_RECEIPT_LINE: int
+A4: tuple[str, str]
+LABELS: dict[str, dict[str, str]]
+SCISSORS_SVG_PATH: str
+
+class Address:
+    @overload
+    @classmethod
+    def create(cls, *, name: str | None = None, line1: str, line2: str | None = None, country: str | None) -> CombinedAddress: ...
+    @overload
+    @classmethod
+    def create(cls, *, name: str | None = None, line1: str | None = None, line2: str, country: str | None) -> CombinedAddress: ...
+    @overload
+    @classmethod
+    def create(
+        cls,
+        *,
+        name: str,
+        street: str | None = None,
+        house_num: str | None = None,
+        pcode: str,
+        city: str,
+        country: str | None = None,
+    ) -> StructuredAddress: ...
+    @staticmethod
+    def parse_country(country: str | None) -> str: ...
+
+class CombinedAddress(Address):
+    combined: Final = True
+    name: str
+    line1: str
+    line2: str
+    country: str
+    def __init__(
+        self, *, name: str | None = None, line1: str | None = None, line2: str | None = None, country: str | None = None
+    ) -> None: ...
+    def data_list(self) -> list[str]: ...
+    def as_paragraph(self, max_chars: int = 72) -> Iterator[str]: ...
+
+class StructuredAddress(Address):
+    combined: Final = False
+    name: str
+    street: str
+    house_num: str
+    pcode: str
+    city: str
+    country: str
+    def __init__(
+        self,
+        *,
+        name: str | None = None,
+        street: str | None = None,
+        house_num: str | None = None,
+        pcode: str | None = None,
+        city: str | None = None,
+        country: str | None = None,
+    ) -> None: ...
+    def data_list(self) -> list[str]: ...
+    def as_paragraph(self, max_chars: int = 72) -> Iterator[str]: ...
+
+class QRBill:
+    qr_type: str
+    version: str
+    coding: int
+    allowed_currencies: tuple[Literal["CHF"], Literal["EUR"]]
+    font_family: str
+    creditor: CombinedAddress | StructuredAddress
+    final_creditor: CombinedAddress | StructuredAddress | None
+    debtor: CombinedAddress | StructuredAddress | None
+    reference_number: str | None
+    @overload
+    def __init__(
+        self,
+        account: str,
+        creditor: _AddressDict,
+        final_creditor: _AddressDict | None = None,
```

Comment:
We could remove `final_creditor` from stubs, because it always raises an error if `final_creditor` is given.

Reply:
Good catch, I think I will just statically set it to `None` for now. The documentation should help people with figuring out why that is the case.